templates/tg-mini-app-seamless-signing-boilerplate-template/README.md

```diff
@@ -0,0 +1,45 @@
+## Overview
+
+This template is made from the [boilerplate template](https://github.com/aptos-labs/create-aptos-dapp/tree/main/templates/boilerplate-template), the difference is that we make this possible to run as a Telegram mini app, and we change the app to be a simple clicker game.
+
+The difference between this template and the [tg-mini-app-boilerplate-template](https://aptos.dev/en/build/create-aptos-dapp/templates/tg-mini-app-boilerplate) is we are using [@mizuwallet-sdk/core](https://docs.mizu.io/docs/api-connect/core) instead of Aptos Wallet Adapter that gives you more control on the wallet flow, this way we achieve a seamless signing experience, i.e. users no longer need to approve the transaction explicitly.
```

Comment:
```suggestion
The difference between this template and the [clicker game Telegram mini app template](https://aptos.dev/en/build/create-aptos-dapp/templates/tg-mini-app-boilerplate) is we are using [@mizuwallet-sdk/core](https://docs.mizu.io/docs/api-connect/core) instead of Aptos Wallet Adapter that gives you more control on the wallet flow, this way we achieve a seamless signing experience, i.e. users no longer need to approve the transaction explicitly.
```